desktop/core/src/desktop/js/apps/admin/ServerLogs/ServerLogsTab.tsx

```diff
@@ -0,0 +1,114 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useState, useEffect } from 'react';
+import { Spin, Alert } from 'antd';
+import ServerLogsHeader from './ServerLogsHeader';
+import useLoadData from '../../../utils/hooks/useLoadData';
+import './ServerLogsTab.scss';
+
+interface ServerLogsData {
+  logs: string[];
+  hue_hostname: string;
+}
+
+const ServerLogs: React.FC = (): JSX.Element => {
+  const [filter, setFilter] = useState<string>('');
+  const [wrapLogs, setWrapLogs] = useState(true);
+
+  const {
+    data: logsData,
+    loading,
+    error
+    //reloadData
+  } = useLoadData<ServerLogsData>('/api/v1/logs');
+
+  useEffect(() => {
+    const updateSize = () => {
+      const newHeight = document.documentElement.clientHeight - 250;
```

Comment:
What is 250 here? Note that we can't rely on positioning from the top of the window as in some cases there's a banner on top.

Reply:
If the intention is to "anchor" some element top and bottom you can use css to achieve the same, for instance absolute positioning.